src/app/method/Equipment.js

```diff
@@ -126,25 +185,36 @@ define([
                 that.onCathodeTypeChange(event.target.value);
             });
 
-            $(this.raftTab).on('show.bs.tab', function setNum() {
-                // don't overrite a value that was set by inprogress cache
-                if (that.numberNettersTxt.value === '') {
-                    that.numberNettersTxt.value = 1;
-                }
-                that.anodeShapeSelect.value = '';
-                $(that.anodeShapeSelect).combobox('toggle');
-            });
-            $(this.raftTab).on('hide.bs.tab', function setNum() {
-                that.numberNettersTxt.value = '';
-            });
-
             query('.nav-pills li', this.domNode).on('click', function (event) {
                 that.typeTxt.value = event.currentTarget.dataset.type;
-                that.cacheInProgressData();
             });
 
             this.inherited(arguments);
         },
+        toggleFields: function (event) {
+            // summary:
+            //      click event handler for toggling the equipment field options
+            // click event
+            console.info('app/method/Equipment:toggleFields', arguments);
+
+            var activeTab = event.target.id.toLowerCase();
+            if (!activeTab) {
+                return;
+            }
+
+            var that = this;
+            setTimeout(function () {
```

Comment:
Would firing this method in response to a [bootstrap tab event](https://getbootstrap.com/docs/3.3/javascript/#tabs-events) remove the need for a settimeout?

Reply:
I think since I removed the tabs and only kept the buttons that those events don't fire.